src/foobar/admin.py

```diff
@@ -192,6 +192,30 @@ def changelist_view(self, request, extra_context=None):
         return response
 
 
+@admin.register(models.WalletLogEntry)
+class WalletLogEntryAdmin(ReadOnlyMixin, admin.ModelAdmin):
+    list_display = ('date_created',
+                    'user',
+                    'trx_type',
+                    'comment',
+                    'pre_balance',
+                    'amount',
+                    'post_balance'
+                    )
+    readonly_fields = list_display
```

Comment:
![image](https://cloud.githubusercontent.com/assets/74944/23169407/f11b7ae2-f84b-11e6-83fb-99ea9f9dd5f5.png)
The `wallet` field is not readonly which means someone could mess with us and attach a log entry to some other wallet. We should either make it read only or ideally simple hide it from the form.